modules/ROOT/pages/manage-backup-restore.adoc

```diff
@@ -1,63 +1,93 @@
+====
 
 
-== Prerequisites for Runtime Fabric on VMs / Bare Metal
+== Prerequisites for Runtime Fabric
 
-Before performing a backup or restore operation, verify that Runtime Fabric on VMs / Bare Metal was installed or upgraded 
-using  https://docs.mulesoft.com/release-notes/runtime-fabric/runtime-fabric-installer-release-notes[installer version 1.1.1581474166, role=external, window=_blank] or later, and that the rtfctl CLI tool is upgraded to version 0.2.95 or later. The versions can be verified by running `rtfctl version` on a Runtime Fabric node.
+Before performing a backup or restore operation:
+
+* Verify that Runtime Fabric is installed and running successfully.
+* Verify that `rtfctl` CLI tool is upgraded to version 0.3.102 or later. Verify the version by running `rtfctl version` on a Runtime Fabric node.
+* If you are using Runtime Fabric on VMs / Bare Metal, verify that Runtime Fabric is upgraded to version 1.1.1581474166 or later. See xref:release-notes::runtime-fabric/runtime-fabric-installer-release-notes.adoc[Anypoint Runtime Fabric Installer Release Notes].
 
 == Create a Backup
 
-To create a backup:
+To create a backup run the following command:
 
-. Run the following command on any controller node in the cluster as a privileged user (with root access):
-+
 ----
 ./rtfctl backup <path_to_backup_file>
 ----
+
 This command creates an archive of the current system state in `<path_to_backup_file>`, which can be any path in the file system that you have write access to, for instance `/opt/anypoint/runtimefabric/backup.tar.gz`. 
 
-. Move the archive file to secure storage (a different machine that is not part of the RTF cluster);
-for example:
-+
+[NOTE]
+====
+For Runtime Fabric running on VMs / Bare Metal, the `rtfctl` binary is present in directory `/opt/anypoint/runtimefabric/`. Confirm that you run the backup command on any controller node in the cluster as a privileged user. To use the archive for restoring later, copy the archive file to secure storage that is on a different machine and is not part of the RTF cluster. The following command, for example, uses `scp` to copy the archive to another drive: 
+
 ----
 scp your_username@remotehost:/opt/anypoint/runtimefabric/backup.tar.gz /backup-path/to-restore.tar.gz
 ----
 
+For Runtime Fabric running on Self-Managed Kubernetes, confirm that the `rtfctl` binary is present in the current directory and Kubernetes(kubectl) context is set to the intended backup cluster.
+====
+
 == Restore a Cluster
 
-To restore a Runtime Fabric cluster from a backup, you can use the existing Runtime Fabric cluster or create a new Runtime Fabric 
-cluster with the same hardware (number of servers, disks, and so on) as the cluster being restored.
+To restore Runtime Fabric cluster from a backup, use an existing Runtime Fabric cluster or create a new Kubernetes cluster with the same configuration (number of servers, disks, and so on) as the backed up cluster.  
```

Comment:
`same configuration (number of servers, disks, and so on) as the backed up cluster. `
is this a hard requirement?

Reply:
perhaps we should also clarify if the cluster being restored to is different, the clutser should be provisioned without an association to a new fabric (i.e. just a gravity cluster without runtime fabric software installed)?

Reply:
@kmendiratta-crm 
- Same configuration is a hard requirement on the gravitational cluster but it should work for Self-managed Kubernetes. But I think this line will provide as a general guideline for backup and restore funcionality.
- This already provided in note section below